### DISCUSSION TITLE
0.69 release is now available

### DISCUSSION THREAD

**Discussion by douglowder** [react-native-tvos/react-native-tvos#356] [Announcements]
The TV release for RN 0.69 has been published.

https://github.com/react-native-tvos/react-native-tvos/releases/tag/v0.69.5-0

**Comment by EyMaddis** [comment]
What is the proper way to include the build hotfix introduced with v0.69.7?
https://github.com/facebook/react-native/releases/tag/v0.69.7

**Reply by douglowder** [reply]
Sorry I did not see this earlier. I will cherry-pick https://github.com/facebook/react-native/commit/c4da74c4636cbbd6bbf681d39a8a8cca49f11f56 to the 0.69.6 release train for this repo and will provide that in the next release.
